docs/contacts/activity/log-meeting.md

```diff
@@ -0,0 +1,67 @@
+# Hubspot: Log a meeting
```

Comment:
```suggestion
# Log a meeting
```

Reply:
I would suggest either adding "Hubspot:" to all of the trainings or removing it from all just for consistency